test/release/examples/primers/chplvis/chplvis.html

```diff
@@ -0,0 +1,312 @@
+<!DOCTYPE HTML>
```

Comment:
At a high level, I think moving this file on to the chapel website will change a lot of its formatting, so that it matches the rest of the content present there.  Someone who is more expert on that area should comment as to how thing will need to change